packages/itinerary-body/src/styled.tsx

```diff
@@ -522,15 +533,21 @@ export const StepDescriptionContainer = styled.div`
   margin-left: 24px;
   line-height: 1.25em;
   padding-top: 1px;
+
+  & > span {
+    margin-right: 1ch;
+  }
+`;
+
+export const StepsHeaderAndMap = styled.span`
```

Comment:
Would it be possible to rename this? This makes it seem like a map is being rendered

Reply:
This `styled` module is exported at package level, so renaming would be a breaking change.

Reply:
> This `styled` module is exported at package level, so renaming would be a breaking change.

Disregard response above.

Reply:
Yes I didn't finish typing the name! Renamed in https://github.com/opentripplanner/otp-ui/pull/395/commits/5ad35a818775c148fb1ce4a7f52c3bb627feb22c.